doc/source/language/variables.rst

```diff
@@ -604,8 +610,10 @@ duration of the entire file's compile.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Normally the keywords ``local`` and ``global`` can be left off
-as optional in declare statements.  But when you turn LAZYGLOBAL
-off, the compiler starts requiring them to be explicitly stated.
+as optional in declare **identifier** statements.  But when you
```

Comment:
Yeah, sort of.  I'm using the boldness to communicate speech emphasis.  In this instance I'm emphasizing the difference between types of declare statement - that it's declare **identifier** as opposed to declare **parameter** or declare **function**.